rep-2010.rst

```diff
@@ -234,7 +234,10 @@ Any existing tooling for aiding users in setting up ROS security should consider
 Command line interface
 ----------------------
 
-TODO: Consider proposing an interface for a CLI here. Although it seems a bit tangent to the core feature, I think it is important to the proposal since it makes it so people can immediately start using the feature.
+We can have a tool like ``ros2 service echo example_interfaces/srv/AddTwoInts`` that internally subscribes to the two hidden topics ``example_interfaces/srv/AddTwoInts_RequestEvent`` and
```

Comment:
Just wondering isn't `example_interfaces/srv/AddTwoInts` the service type I think, we would like have the service name instead maybe in `echo`?

Reply:
Updated